FileLiberator/FileLiberator.csproj

```diff
@@ -13,4 +13,10 @@
     <ProjectReference Include="..\InternalUtilities\InternalUtilities.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Reference Include="taglib-sharp">
+      <HintPath>DecryptLib\taglib-sharp.dll</HintPath>
```

Comment:
Obviously there's been a lot of churn lately so I want to clarify: this PR removed AtomicParsley and postproc-54.dll from the decrypter and added taglib-sharp.dll to the domain aware FileLiberator. I imagined FileLiberator going through the domain ignorant decrypter to do this.